flexget/plugins/metainfo/nfo_lookup.py

```diff
@@ -104,6 +108,13 @@ def get_nfo_filename(self, entry):
         if os.path.isfile(nfo_full_filename):
             return nfo_full_filename
 
+    @staticmethod
+    def is_valid_imdb_title_id(value=None):
```

Comment:
I didn't want to touch code in other files, but that was were I first looked to see if such a function existed. I'll move the function to there and create a similar one for "person ID".